### PR TITLE
Add bilibili.tv presence

### PR DESCRIPTION
**Pull request by epiceason**
Images:
video
anime

### CODE REVIEW

File: websites/B/bilibilitv/presence.ts [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    }  else {
+        presenceData.details = "Viewing dashboard";
+        presenceData.state = "Bilibili Studio";
+    }
+  }
+  
+  if (!presenceData.details) {
+    presence.setTrayTitle();
+    presence.setActivity();
+  } else presence.setActivity(presenceData);
```
Comment (EncryptedDev) [inline websites/B/bilibilitv/presence.ts:106]:
```suggestion
if (presenceData.details) presence.setActivity(presenceData);
else presence.setActivity();
```

**Review by Bas950 — CHANGES_REQUESTED** [review]
Should be added / fixed in https://github.com/PreMiD/Presences/tree/master/websites/B/bilibili

**Comment by easonuwu** [comment]
Its a different site so, i dont think it should be merged together

**Review by Bas950 — DISMISSED** [review]

File: websites/B/bilibilitv/presence.ts [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+    } else {
+        presenceData.details = "Viewing dashboard";
+        presenceData.state = "Bilibili Studio";
+    }
+  }
+  
+  if (!presenceData.details) {
+    presence.setTrayTitle();
+    presence.setActivity();
+  } else presence.setActivity(presenceData);
```
Comment (EncryptedDev) [inline websites/B/bilibilitv/presence.ts:102]:
```suggestion
if (presenceData.details) presence.setActivity(presenceData);
  else presence.setActivity();
```

**Review by EncryptedDev — APPROVED** [review]

**Review by Slowlife01 — APPROVED** [review]

**Pull request merged**